### PR TITLE
Add `:::ecmascript-algorithm` wrapper with styles

### PR DESCRIPTION
**Pull request by mathiasbynens**
Ref. https://github.com/v8/v8.dev/pull/331#discussion_r380717839.

cc @marjakh

### CODE REVIEW

File: src/blog/understanding-ecmascript-part-1.md [hunk 1]
```diff
@@ -117,29 +123,42 @@ If an algorithm throws an exception, it means returning a Completion Record with
 
 [`ReturnIfAbrupt(argument)`](https://tc39.es/ecma262/#sec-returnifabrupt) means taking the following steps:
 
+:::ecmascript-algorithm
+<!-- markdownlint-disable blanks-around-lists -->
```
Comment (mathiasbynens) [inline src/blog/understanding-ecmascript-part-1.md]: We only need these when `:::ecmascript-algorithm` contains nothing but a list.

Comment (RReverser) [reply src/blog/understanding-ecmascript-part-1.md]: Is there any way to configure markdownlint to ignore all these instances?

Comment (mathiasbynens) [reply src/blog/understanding-ecmascript-part-1.md]: Not that I know of. I don't think we want it for all of them though.

Comment (RReverser) [reply src/blog/understanding-ecmascript-part-1.md]: Okay, let's go with this for now, we can always iterate later.

**Comment by RReverser** [comment]
Thanks @mathiasbynens!

**Pull request merged**